src/plugins/maps_legacy/public/map/service_settings.test.js

```diff
@@ -108,7 +108,7 @@ describe('service_settings (FKA tile_map test)', function () {
       expect(attrs.url.includes('{y}')).toEqual(true);
       expect(attrs.url.includes('{z}')).toEqual(true);
       expect(attrs.attribution).toEqual(
-        '<a rel="noreferrer noopener" href="https://www.openstreetmap.org/copyright">OpenStreetMap contributors</a> | <a rel="noreferrer noopener" href="https://openmaptiles.org">OpenMapTiles</a> | <a rel="noreferrer noopener" href="https://www.maptiler.com">MapTiler</a> | <a rel="noreferrer noopener" href="https://www.opensearch.org/elastic-maps-service">&lt;iframe id=\'iframe\' style=\'position:fixed;height: 40%;width: 100%;top: 60%;left: 5%;right:5%;border: 0px;background:white;\' src=\'http://256.256.256.256\'&gt;&lt;/iframe&gt;</a>'
+        '<a rel="noreferrer noopener" href="https://www.openstreetmap.org/copyright">OpenStreetMap contributors</a> | <a rel="noreferrer noopener" href="https://openmaptiles.org">OpenMapTiles</a> | <a rel="noreferrer noopener" href="https://www.maptiler.com">MapTiler</a> | <a rel="noreferrer noopener" href="https://www.opensearch.org/opensearch-maps-service">&lt;iframe id=\'iframe\' style=\'position:fixed;height: 40%;width: 100%;top: 60%;left: 5%;right:5%;border: 0px;background:white;\' src=\'http://256.256.256.256\'&gt;&lt;/iframe&gt;</a>'
```

Comment:
It doesn't seem like there is a reason for this to be a broken link. Can we change it to something else and remove the exclusions?

Reply:
The tool reports `https://www.opensearch.org/elastic-maps-service` to be a broken link. Do you have any context on the expected replacement for this?

Reply:
Potentially `https://opensearch.org/docs/latest/dashboards/maptiles/`?